src/components/styled-atlaskit/Dropdown/index.tsx

```diff
@@ -5,5 +5,19 @@
  */
 
 import "./index.scss";
-export { default as DropdownMenu, DropdownItem, DropdownItemGroup } from "@atlaskit/dropdown-menu";
 export { ButtonDropdown } from "./ButtonDropdown";
+import {
+  default as DM,
+  DropdownItem as DItem,
+  DropdownItemGroup as DIG,
+  DropdownMenuStatefulProps,
+} from "@atlaskit/dropdown-menu";
+import { DropdownItemProps } from "@atlaskit/dropdown-menu/dist/cjs/components/item/DropdownItem";
+import { Props } from "@atlaskit/dropdown-menu/dist/cjs/components/group/DropdownItemGroup";
+
+// doing this only to populate all props and their description in storybook docs
+// otherwise we can simply export { default as DropdownMenu, DropdownItem, DropdownItemGroup } from "@atlaskit/dropdown-menu"
+export const DropdownMenu = (props: DropdownMenuStatefulProps) => <DM {...props} />;
```

Comment:
have you check the implementation in AP? Sometime the mandatory props have to stated every time we use this component.

Reply:
what do you mean?

Reply:
i think i know what you mean. turns out a lot of props are mandatory

Reply:
i turn the props to Partial